docs/dev/zksync-v2/bridging-funds.md

```diff
@@ -6,11 +6,6 @@ Bridging is implemented by having two contracts
 (one deployed to L1, and the second deployed to L2)
 communicating with each other using [L1 <-> L2 interoperability](./l1-l2-interop.md).
 
-There are two dedicated transaction types for native token bridging:
```

Comment:
what are these replaced by?

Reply:
There is currently a single transaction type: Execute, i.e. a normal contract call. Bridging is implemented with these & L1<>L2 messaging. There is no magic in bridging since the last regenesis, it was an artifact from the first testnets